cmd/dashboard.go

```diff
@@ -0,0 +1,23 @@
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/dapr/dashboard/pkg/version"
+)
+
+// RunDashboard runs the dashboard with the supplied flags
+func RunDashboard() {
```

Comment:
We no need to expose running dashboard  version..Just show the current dashboard version in the Overview page...